public/editor/scripts/main.js

```diff
@@ -52,6 +52,16 @@ require(["jquery", "bowser"], function($, bowser) {
 
   Bramble.once("error", onError);
 
+  var errorMessageTimeoutMS = 10000;
```

Comment:
I think 10s is too short.  I'd do 15s.  We don't want this coming up on slow computers/networks.